### ISSUE TITLE
Allow consuming Winglibs from JSII-supported languages

### ISSUE THREAD

**Issue by eladb** [winglang/wing#6904]
### Use Case

I'd like to consume Winglibs from Python, Java, Golang and .NET.

The top priority is to support preflight APIs.

### Proposed Solution

Generate .jsii manifest when compiling a Winglib and then use JSII to produce language bindings.

This will only work for preflight code, but may be good enough for many use cases.

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.
* If this issue is labeled **needs-discussion**, it means the spec has not been finalized yet. Please reach out on the #dev channel in the [Wing Discord](https://t.winglang.io/discord).

**Comment by gofri** [comment]
Sharing my use case:
I need to set up a very common type of LLM pipeline in Python.
If I could call/execute Wing from in Python via any reasonable API, it would be my go-to for the project. Especially because it means I can run locally & remotely without writing extra code.

**Comment by eladb** [comment]
@gofri would it make sense to stand up a `cloud.Api` wing endpoint and call it via a simple http request?

**Comment by gofri** [comment]
@eladb 
Yes, I think that would be a good step forward, and I can wrap it with some RPC.
I'm not sure if py-to-js interop libs would work with Wing, but that would probably save a lot of boilerplate for that kind of server.
